profiles/common/modules/features/multisite_settings/multisite_settings_core/multisite_settings_core.install

```diff
@@ -37,38 +55,13 @@ function multisite_settings_core_update_7121() {
   $vids = variable_get('taxonomy_browser_vocabulary');
   $vids[$tags_vocab->vid] = $tags_vocab->vid;
   variable_set('taxonomy_browser_vocabulary', $vids);
+  multisite_drupal_toolbox_add_block_context('workbench_moderate_all', 'taxonomy_browser-taxonomy_browser_vocabulary_' . $tags_vocab->vid, 'taxonomy_browser', 'taxonomy_browser_vocabulary_' . $tags_vocab->vid, 'sidebar_left');
 
-  $context = new stdClass();
-  $context->disabled = FALSE; /* Edit this to true to make a default context disabled initially */
-  $context->api_version = 3;
-  $context->name = 'workbench_moderate_all';
-  $context->description = '';
-  $context->tag = 'workbench';
-  $context->conditions = array(
-    'path' => array(
-      'values' => array(
-        'admin/workbench/moderate-all*' => 'admin/workbench/moderate-all*',
-      ),
-    ),
-  );
-  $context->reactions = array(
-    'block' => array(
-      'blocks' => array(
-        'taxonomy_browser-taxonomy_browser_vocabulary_' . $tags_vocab->vid => array(
-          'module' => 'taxonomy_browser',
-          'delta' => 'taxonomy_browser_vocabulary_' . $tags_vocab->vid,
-          'region' => 'sidebar_left',
-          'weight' => '-10',
-        ),
-      ),
-    ),
-  );
-  $context->condition_mode = 0;
-
-  context_save($context);
 }
 
 /**
+ * Implements hook_update_N().
```

Comment:
No need for this: https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#updateN